app/helpers/eq.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
+const eq = (params) => params[0] === params[1];
```

Comment:
This isn't necessary. Looking through the package.json, we already have ember-truth-helpers installed https://github.com/CenterForOpenScience/ember-preprints/blob/develop/package.json#L68 so you can just use that instead of defining your own.